tests/phpunit/Configuration/ConfigurationAssertions.php

```diff
@@ -94,7 +93,7 @@ private function assertConfigurationStateIs(
             $expectedPhpUnit->getConfigDir(),
             $expectedPhpUnit->getCustomPath()
         );
-        $this->assertSame($expectedMutators, $configuration->getMutators());
+        $this->assertEqualsWithDelta($expectedMutators, $configuration->getMutators(), 10.);
```

Comment:
Not cool with that but the only alternative is allowing `assertEquals()`. `assertSame()` cannot work here since the mutators are different.

Another alternative is to have a dedicated assertion for those, but it's a lot of work just for that at this point so I did not pursue this idea